docs/compression_algorithms/Pruning.md

```diff
@@ -69,6 +69,18 @@ and reduce the corresponding accuracy drop even before model training. This opti
 sparsity and filter pruning algorithms. It can be enabled by setting a non-zero value of `num_bn_adaptation_samples` in the
 `batchnorm_adaptation` section of the `initializer` configuration (see example below).
 
+#### Interlayer ranking types
```

Comment:
@AlexKoff88 Please, review this description and suggest whether a more detailed description of the algorithm or a link to the article is necessary